nexus/src/app/sagas/instance_migrate.rs

```diff
@@ -612,6 +613,7 @@ mod tests {
                 external_ips: vec![],
                 disks: vec![],
                 boot_disk: None,
+                cpu_platform: None,
```

Comment:
sure is weird that this file doesn't include a change to `SledReservationConstraintBuilder` like in `instance_start.rs` huh?

that's a bug. this currently would allow a Turin to migrate onto a Milan (bad)